Gemfile

```diff
@@ -1,7 +1,3 @@
 source "https://rubygems.org"
 
-# runtime dependencies from gemspec
-gemspec = File.read("cocoaseeds.gemspec")
-gemspec.scan(/(?<=s\.add_runtime_dependency ).*/).each do |dependency|
-  eval "gem #{dependency}"
-end
+gemspec
```

Comment:
It's the recommended way to include run time and delelopment dependecies for local development of the gem. 

From the [bundler docs](https://bundler.io/v1.14/guides/creating_gem.html)

> Gemfile: Used to manage gem dependencies for our library’s development. This file contains a gemspec line meaning that Bundler will include dependencies specified in foodie.gemspec too. It’s best practice to specify all the gems that our library depends on in the gemspec.

I also added `rake` as a development dependency since it's used to run the tests